Makefile

```diff
@@ -319,6 +319,12 @@ ${ISTIO_OUT}/archive: istioctl-all LICENSE README.md istio.VERSION install/updat
                                  -r "$(VERSION)" -E "$(ISTIO_URL)/deb" -d "${ISTIO_OUT}/archive"
 	release/create_release_archives.sh -v "$(VERSION)" -o "${ISTIO_OUT}/archive"
 
+# istioctl-install builds then installs istioctl into $GOPATH/BIN
```

Comment:
I think but build already creates istioct - just add the new bin directory to your path ( and make sure you set ISTIO_BIN env, tests are using it )

Reply:
Build does but you have to rebuild everything each time you want to verify your code works (the feedback loop is not fast). I only added this because I found myself re-writing it every time I wanted to do any dev work on istioctl.